recipes/geotexxx/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "geotexxx" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/geotexxx-{{ version }}.tar.gz
+  sha256: 0ca0c47e8760e8a2d0ebeb08956aa4674daa3a0ee2870e361c7ecbba5fa70538
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

This looks like it's pure Python and could be noarch:
```suggestion
build:
  script: {{ PYTHON }} -m pip install . -vv
  number: 0
  noarch: python
```